qiskit_ibm_runtime/transpiler/passes/scheduling/dynamical_decoupling.py

```diff
@@ -184,14 +186,20 @@ def __init__(
                 for execution on large backends.
             dd_barrier: only apply DD to delays terminating with a barrier
                 whose label contains the specified string
+            block_ordering_callable: A callable used to produce an ordering of the nodes to minimize the
+                number of blocks needed. If not provided, a potentially slow but performant algorithm is
+                used.
```

Comment:
(comment applies to all similar docstrings)
I'd just link to the default function, rather than trying to pass judgement on it :)

Reply:
Done :)